### PR TITLE
Added an initial bunch of properties for electric motors

### PR DESCRIPTION
**Pull request by dakrbo**
Signed-off-by: Daniel (dakrbo) <daniel.krippner@de.bosch.com>

### CODE REVIEW

File: spec/Powertrain/PowerSource/ElectricMotor.vspec [hunk 1]
```diff
@@ -10,3 +11,99 @@
...
+
+
+#
+# Motor rotations per minute
+#
+- Motor.Speed:
+  datatype: int16
+  type: sensor
+  unit: rpm
+  min: 0
```
Comment (chheis) [inline spec/Powertrain/PowerSource/ElectricMotor.vspec]: min shall be possible to be negative: min: -100000

File: spec/Powertrain/PowerSource/ElectricMotor.vspec [hunk 2]
```diff
@@ -10,3 +11,99 @@
...
+
+
+#
+# Current Power
+#
+- Motor.Power:
+  datatype: int16
+  type: sensor
+  unit: kW
+  min: 0
```
Comment (chheis) [inline spec/Powertrain/PowerSource/ElectricMotor.vspec]: min shall be possible to be negative: min: -2000

File: spec/Powertrain/PowerSource/ElectricMotor.vspec [hunk 3]
```diff
@@ -10,3 +11,99 @@
...
+  description: Current motor power output. Negative values indicate regen mode.
+
+#
+# Current Torque
+#
+- Motor.Torque:
+  datatype: int16
+  type: sensor
+  unit: N.m
+  min: 0
```
Comment (chheis) [inline spec/Powertrain/PowerSource/ElectricMotor.vspec]: min shall be possible to be negative: min: -5000

**Comment by dakrbo** [comment]
Thanks @chheis !

For the benefit of everyone else: Christian helped me with our Bosch-prescribed review here, because I was to fast on the 'create PR' button in our project-fork...

File: spec/Powertrain/PowerSource/ElectricMotor.vspec [hunk 4]
```diff
@@ -10,3 +11,99 @@
...
+#
+- Motor:
+  type: branch
+  description: motor signals
+
+
+#
+# Motor rotations per minute
+#
+- Motor.Speed:
```
Comment (adobekan) [inline spec/Powertrain/PowerSource/ElectricMotor.vspec:54]: RotationalSpeed?

Comment (gunnarx) [reply spec/Powertrain/PowerSource/ElectricMotor.vspec:54]: Agree, it should be something different than Speed.

Comment (chheis) [reply spec/Powertrain/PowerSource/ElectricMotor.vspec:54]: Maybe direclty "rpm" ?

Comment (gunnarx) [reply spec/Powertrain/PowerSource/ElectricMotor.vspec:54]: Motor.RPM would also be acceptable for me.  (I think "RPM" it is very well known concept)

Comment (SebastianSchildt) [reply spec/Powertrain/PowerSource/ElectricMotor.vspec:54]: I also like RPM. Maybe later we should give some love to spec/Powertrain/PowerSource/CombustionEngine.vspec  to make it consistent

File: spec/Powertrain/PowerSource/ElectricMotor.vspec [hunk 5]
```diff
@@ -10,3 +11,99 @@
...
+  type: attribute
+  default: 0
+  unit: kW
+  description: Peak power, in kilowatts, that motor(s) can generate.
+
+- MaxTorque:
+  datatype:  uint16
+  type: attribute
+  default: 0
+  unit: N.m
```
Comment (SebastianSchildt) [inline spec/Powertrain/PowerSource/ElectricMotor.vspec:25]: N.m or Nm?

Comment (gunnarx) [reply spec/Powertrain/PowerSource/ElectricMotor.vspec:25]: vss-tools defines "N.m", but that seems wrong.  Let's open a ticket to fix it.

Comment (SebastianSchildt) [reply spec/Powertrain/PowerSource/ElectricMotor.vspec:25]: Clean up later: https://github.com/GENIVI/vehicle_signal_specification/issues/173

**Pull request merged**